### PR TITLE
[Doc] add 2.3.15&16 rn

### PR DESCRIPTION
**Pull request by hellolilyliuyi**
Fixes #issue

## What type of PR is this:
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

## Checklist:
- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

File: docs/release_notes/release-2.3.md [hunk 1]
```diff
@@ -1,5 +1,39 @@
...
+
+- Optimized tablet scheduling logic to prevent tablets remaining pending for a long period or an FE crashing under certain circumstances. [#21647](https://github.com/StarRocks/starrocks/pull/21647) [#23062](https://github.com/StarRocks/starrocks/pull/23062) [#25785](https://github.com/StarRocks/starrocks/pull/25785)
+- Optimized the scheduling logic of TabletChecker to prevent the checker from repeatedly scheduling tablets that are not repaired. [#27648](https://github.com/StarRocks/starrocks/pull/27648)
+- The partition metadata records visibleTxnId, which corresponds to the visible version. When versions of replicas for a tablet are not consistent, it is easier to trace the transaction that created this version . [#27924](https://github.com/StarRocks/starrocks/pull/27924)
+
+### Bug Fixes
+
+Fixed the following issue:
+
+- Incorrect table-level scan statistics in FE cause inaccurate metrics for table queries and loading. [#28022](https://github.com/StarRocks/starrocks/pull/28022)
```
Comment (amber-create) [inline docs/release_notes/release-2.3.md:27]: image
-> the FEs

File: docs/release_notes/release-2.3.md [hunk 2]
```diff
@@ -1,5 +1,39 @@
...
+
+## Improvements
+
+- Optimized tablet scheduling logic to prevent tablets remaining pending for a long period or an FE crashing under certain circumstances. [#21647](https://github.com/StarRocks/starrocks/pull/21647) [#23062](https://github.com/StarRocks/starrocks/pull/23062) [#25785](https://github.com/StarRocks/starrocks/pull/25785)
+- Optimized the scheduling logic of TabletChecker to prevent the checker from repeatedly scheduling tablets that are not repaired. [#27648](https://github.com/StarRocks/starrocks/pull/27648)
+- The partition metadata records visibleTxnId, which corresponds to the visible version. When versions of replicas for a tablet are not consistent, it is easier to trace the transaction that created this version . [#27924](https://github.com/StarRocks/starrocks/pull/27924)
+
+### Bug Fixes
+
+Fixed the following issue:
```
Comment (amber-create) [inline docs/release_notes/release-2.3.md:25]: issue -> issues

File: docs/release_notes/release-2.3.md (hunk 1 above)
Comment (amber-create) [inline docs/release_notes/release-2.3.md:27]: for -> related to

File: docs/release_notes/release-2.3.md [hunk 3]
```diff
@@ -1,5 +1,39 @@
...
+
+- Incorrect table-level scan statistics in FE cause inaccurate metrics for table queries and loading. [#28022](https://github.com/StarRocks/starrocks/pull/28022)
+- BEs may crash if the Join key is a large BINARY column. [#25084](https://github.com/StarRocks/starrocks/pull/25084)
+- An aggregate operator may trigger thread safety issues in certain scenarios, causing BEs to crash. [#26092](https://github.com/StarRocks/starrocks/pull/26092)
+- The version number for a tablet is inconsistent between the BE and FE after data is restored. [#26518](https://github.com/StarRocks/starrocks/pull/26518/files)
+- Partitions can not be automatically created for recovered tables. [#26813](https://github.com/StarRocks/starrocks/pull/26813)
```
Comment (amber-create) [inline docs/release_notes/release-2.3.md:31]: can not -> cannot

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by amber-create — DISMISSED** [review]

File: docs/release_notes/release-2.3.md [hunk 4]
```diff
@@ -1,5 +1,39 @@
...
+
+## 2.3.15
+
+Release date: July 31, 2023
+
+## Improvements
+
+- Optimized tablet scheduling logic to prevent tablets remaining pending for a long period or an FE crashing under certain circumstances. [#21647](https://github.com/StarRocks/starrocks/pull/21647) [#23062](https://github.com/StarRocks/starrocks/pull/23062) [#25785](https://github.com/StarRocks/starrocks/pull/25785)
+- Optimized the scheduling logic of TabletChecker to prevent the checker from repeatedly scheduling tablets that are not repaired. [#27648](https://github.com/StarRocks/starrocks/pull/27648)
+- The partition metadata records visibleTxnId, which corresponds to the visible version. When versions of replicas for a tablet are not consistent, it is easier to trace the transaction that created this version . [#27924](https://github.com/StarRocks/starrocks/pull/27924)
```
Comment (evelynzhaojie) [inline docs/release_notes/release-2.3.md:21]: 1. consistent between who?  suggest: The partition metadata records visibleTxnId, which corresponds to the visible version of tablet replicas. When the replica version of a tablet on the BE is inconsistent with that on the FE, xxx      2. extra space at the end of this sentence

File: docs/release_notes/release-2.3.md [hunk 5]
```diff
@@ -1,5 +1,39 @@
...
+- The version number for a tablet is inconsistent between the BE and FE after data is restored by using [RESTORE](../sql-reference/sql-statements/data-definition/RESTORE.md). [#26518](https://github.com/StarRocks/starrocks/pull/26518/files)
+- Partitions cannot be automatically created after the table is recovered by using [RECOVER](../sql-reference/sql-statements/data-definition/RECOVER.md). [#26813](https://github.com/StarRocks/starrocks/pull/26813)
+- The loading transaction remains in a pending state and the DDL statements hang when the strict mode is enabled for loading data by using INSERT INTO, and data to be loaded do not satisfy the quality requirement.
+- Some INSERT jobs return `[42000][1064] Dict Decode failed, Dict can't take cover all key :0` if low-cardinality optimization is enabled. [#27395](https://github.com/StarRocks/starrocks/pull/27395)
+- In certain cases, the INSERT INTO SELECT operation times out when the Pipeline is not enabled.
+- The query returns no data when the query condition is `WHERE partition_column < xxx` and the value in `xxx` is only premise to hour, not to minute and second, for example, `2023-7-21 22`.
```
Comment (evelynzhaojie) [inline docs/release_notes/release-2.3.md:35]: "premise" is the wrong word，i think you want to say "precise“，change to "is only accurate to hour"

File: docs/release_notes/release-2.3.md [hunk 6]
```diff
@@ -1,5 +1,39 @@
...
+### Bug Fixes
+
+Fixed the following issues:
+
+- Incorrect table-level scan statistics in the FEs cause inaccurate metrics related to table queries and loading. [#28022](https://github.com/StarRocks/starrocks/pull/28022)
+- BEs may crash if the Join key is a large BINARY column. [#25084](https://github.com/StarRocks/starrocks/pull/25084)
+- An aggregate operator may trigger thread safety issues in certain scenarios, causing BEs to crash. [#26092](https://github.com/StarRocks/starrocks/pull/26092)
+- The version number for a tablet is inconsistent between the BE and FE after data is restored by using [RESTORE](../sql-reference/sql-statements/data-definition/RESTORE.md). [#26518](https://github.com/StarRocks/starrocks/pull/26518/files)
+- Partitions cannot be automatically created after the table is recovered by using [RECOVER](../sql-reference/sql-statements/data-definition/RECOVER.md). [#26813](https://github.com/StarRocks/starrocks/pull/26813)
+- The loading transaction remains in a pending state and the DDL statements hang when the strict mode is enabled for loading data by using INSERT INTO, and data to be loaded do not satisfy the quality requirement.
```
Comment (evelynzhaojie) [inline docs/release_notes/release-2.3.md:32]: do --- does

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by evelynzhaojie — DISMISSED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by amber-create — APPROVED** [review]

**Review by EsoragotoSpirit — APPROVED** [review]

**Pull request merged**